### PR TITLE
Add media

### PR DESCRIPTION
**Pull request by Karol-Maximus**
Add translations for media page also updated landing page translations.

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -129,6 +129,62 @@ en:
...
+          item5: "• promoting and implementing the idea of restorative justice and strengthening the methods of exercising power."
+        y2015: "Years 2015 - 2017"
+        content1: "In the years 2016-2020, the Association established cooperation with professional lobbyists representing the interests of citizens and our ideas in the Polish parliament. Thus, we participated in the work of parliamentary committees, the direct result of which was the regulation of the Minister of Health allowing research on non-fibrous hemp."
+        content2: "Since 2017, we have been publishing the Index Scientia Cannabis magazine, edited by Dr. Piotr Grzegorczyk, in which we discuss the issues of legal regulations regarding the possession, provision, cultivation and processing of cannabis. As well as monitoring cannabis trading, social and psychological assessment of the determinants of cannabis consumption."
+        content3: "At the same time, we built the first version of the internet platform, on which our Associates could obtain the necessary information on the current activities, as well as obtain advice on the subject of applicable law."
```
Comment (wolak88) [inline config/locales/en.yml:146]: I think those should be indented two more spaces, the same applies for keys: `y2018`, `y2020` and `y2021`

File: config/locales/en.yml [hunk 2]
```diff
@@ -129,6 +129,62 @@ en:
...
+        content3: "The greatest success, however, is joining forces with MGC Pharmaceuticals from Australia, which deals with the development and research of cannabis products, and the introduction to the market of innovative solutions in the field of medical products."
+        y2021: "Year 2021"
+        content1: "In January 2021, the third edition of the CannActive conference took place, entitled: CANNACTIVE.SCIENCE - SCIENCE, MEDICINE OR BUSINESS? The conference was held online for the first time, but it was nevertheless very fruitful in terms of conclusions and solutions for science and business."
+        content2: "We have also accelerated the work on a new IT system to support the respondent as part of the research so that its main modules are ready by the end of June 2021."
+        content3: "In April 2021, we obtained a positive opinion from the Bioethics Committee and we are currently in the preliminary phase of the study. We activate the infrastructure, send out information materials and recruit for research. We invite you to apply!"
+      mentiones: "Mentions about us on other sites"
```
Comment (wolak88) [inline config/locales/en.yml:158]: `mentions`

File: config/locales/en.yml [hunk 3]
```diff
@@ -129,6 +129,62 @@ en:
...
+        a7: "A: These are unique on a global scale pioneering nationwide research. We require responsible citizenship, and hence, among others: registration for real data, periodic completion of research surveys or medical visits."
+        q8: "Q: How can you support the Association?"
+        a8: "A: Volunteering for the Association (lending their talents and skills); Paying the membership fee (becoming an Active Member), making a donation to the Association's bank account: "
+      social: "Social media"
+        link1: "facebook: "
+        link2: "instagram: "
+      team: "Our team"
+      video: "Videos"
+        link1:
+        link2:
```
Comment (wolak88) [inline config/locales/en.yml:183]: I don't think we need to translate the links, though if we want to put different videos please leave it as empty string `""`

File: config/locales/pl.yml [hunk 4]
```diff
@@ -98,6 +98,64 @@ pl:
...
+        content3: "Za największy sukces jednak możemy uznać połączenie sił z MGC Pharmaceuticals z Australii, które zajmuje się opracowywaniem i badaniami nad produktami z konopi, oraz wprowadzaniem na rynek nowatorskich rozwiązań w dziedzinie produktów medycznych."
+        y2021: "Rok 2021"
+        content1: "W styczniu 2021 roku odbyła się trzecia edycja konferencji CannActive pod tytułem: CANNACTIVE.SCIENCE - NAUKA, MEDYCYNA CZY BIZNES? Konferencja pierwszy raz odbyła się on-line lecz mimo to była bardzo owocna we wnioski i rozwiązania dla nauki i biznesu."
+        content2: "Przyspieszyliśmy też prace nad nowym systemem informatycznym do obsługi badanego w ramach prowadzonych badań by jego główne moduły były gotowe na koniec czerwca 2021 roku."
+        content3: "W kwietniu 2021 uzyskaliśmy pozytywną opinię Komisji Bioetycznej i aktualnie znajdujemy się we wstępnej fazie badania. Aktywujemy infrastrukturę, rozsyłamy materiały informacyjne i prowadzmy nabór do badania. Zapraszamy do aplikowania!"
+      mentiones: "Wzmianki o nas na innych stronach"
```
Comment (wolak88) [inline config/locales/pl.yml:127]: Same comments as in `en.yml` file

File: config/locales/en.yml [hunk 5]
```diff
@@ -82,27 +82,27 @@ en:
...
-        content: "Cannabis House gives everyone the chance to help advance the field of medicine and participate in studies at the edge of the scientific field."
-        cta: "Learn more"
+        title: "Help save lives"
+        content: "With its activities, Cannabis House pushes the boundaries of what is possible in the world of cannabis and gives you the opportunity to contribute to progress in the field of medicine. Show your support for our ideas and take part in research!"
+        cta: "Join us!"
       description:
-        title: "We are scientific study fascillitators"
-        content: "Our platform provides a safe space for doctors, researchers, and research participants to collaborate on studying the effects and benefits of medicinal herbs."
+        title: "We are the initiators of scientific research"
+        content: "Our platform for the next few years will provide a safe space for information exchange and collaboration in researching the effects and benefits of cannabis use. It is designed for research participants as well as doctors, researchers and entrepreneurs."
```
Comment (wolak88) [inline config/locales/en.yml:90]: For the next few years our platform will provide..

File: config/locales/en.yml [hunk 6]
```diff
@@ -82,27 +82,27 @@ en:
...
         secondStep:
-          title: "Schedule Appointment"
-          content: "Use our platform to schedule an appointment with the doctor who will approve your profile"
+          title: "Plan your visit"
+          content: "Use our platform to make an appointment with a doctor who will approve your profile"
         thirdStep:
-          title: "Visit Dispensery"
-          content: "With your CH account, you can visit your assigned local dispensery to pick up the study material"
+          title: "Visit the facility"
+          content: "Using your CH account, you can choose a designated local site to collect your research material"
```
Comment (wolak88) [inline config/locales/en.yml:102]: I don't think a user will be choosing the dispensary, so:
`With your CH account you can visit your designated dispensary and collect your research material`

**Review by wolak88 — CHANGES_REQUESTED** [review]
I've added some quick comments

File: config/locales/en.yml (hunk 5 above)
Comment (Karol-Maximus) [reply config/locales/en.yml:90]: Super! I'm changing it.

File: config/locales/en.yml (hunk 1 above)
Comment (Karol-Maximus) [reply config/locales/en.yml:146]: done

File: config/locales/en.yml (hunk 2 above)
Comment (Karol-Maximus) [reply config/locales/en.yml:158]: changed

File: config/locales/en.yml (hunk 3 above)
Comment (Karol-Maximus) [reply config/locales/en.yml:183]: done

File: config/locales/pl.yml (hunk 4 above)
Comment (Karol-Maximus) [reply config/locales/pl.yml:127]: done

File: config/locales/en.yml (hunk 6 above)
Comment (Karol-Maximus) [reply config/locales/en.yml:102]: done

**Comment by Karol-Maximus** [comment]
I'm pushing a new version soon.

**Pull request closed without merging**